src/uwp/Microsoft.Build.Net.CoreRuntimeTask/ResourceHandlingTask.cs

```diff
@@ -295,61 +415,70 @@ private void WriteResW(Stream stream, string reswFilePath)
             }
         }
 
-        private Stream ExtractFromAssembly(string assemblyFilePath, out string assemblyName, out bool containsResources)
+        private List<ResWInfo> ExtractAssemblyResWList(string assemblyFilePath, out bool containsFrameworkResources)
         {
-            assemblyName = null;
-            containsResources = true;
+            containsFrameworkResources = false;
 
             IAssembly assembly = _host.LoadUnitFrom(assemblyFilePath) as IAssembly;
             if (assembly == null || assembly == Dummy.Assembly)
             {
-                containsResources = false;
                 return null;
             }
 
             if (assembly.Resources == null)
             {
-                containsResources = false;
                 return null;
             }
 
-            assemblyName = assembly.Name.Value;
-            string resourcesName = "FxResources." + assemblyName + ".SR.resources";
-            int resourceCount = 0;
+            string neutralResourceLanguage;
+            if (String.IsNullOrEmpty(assembly.Culture) && !TryGetNeutralResourcesLanguageAttribute(assembly, out neutralResourceLanguage))
+            {
+                // Must have NeutralResourcesLanguageAttribute
+                // warning MSB3817: The assembly "C:\Users\zamont\source\repos\08_22_ResGenBeforeAttempt_CSharpApp\ClassLibrary1\bin\x86\Debug\ClassLibrary1.dll" does not have a NeutralResourcesLanguageAttribute on it. To be used in an app package, portable libraries must define a NeutralResourcesLanguageAttribute on their main assembly (ie, the one containing code, not a satellite assembly).
+                return null;
```

Comment:
should we produce the same warning as VS used to do?

Reply:
We should but I don't think I have time to do this completely. We need to check this in asap. I've edited the code to remove my PII.